Misc/NEWS.d/next/Library/2018-07-29-12-14-54.bpo-34226.BE7zbu.rst

```diff
@@ -0,0 +1 @@
+Fix cgi.parse_multipart without content_length
```

Comment:
Use formatting for cgi.parse_multipart:

cgi.parse_multipart -> ``cgi.parse_multipart``.

Also, add "Patch by Roger Duran" at the end.

Reply:
Ok, fixed